docs/user_guide/plugins/polygon_encoded.md

```diff
@@ -0,0 +1,19 @@
+```{code-cell} ipython3
+---
+nbsphinx: hidden
+---
+import folium
+from folium import plugins
+```
+
+# PolygonFromEncoded
+
```

Comment:
I'd expect this documentation to have one or two sentences that explain what this class is. Like the headline of the docstring "Create PolyLines directly from the encoded string" or something like that.

And maybe more importantly, something about, or a link to, how that encoding works.

Reply:
done